bentoml/_internal/bento/docker/templates/base.j2

```diff
@@ -56,10 +56,18 @@ COPY --chown={{ bento__user }}:{{ bento__user }} . ./
 
 {% set __install_python_scripts__ = expands_bento_path("env", "python", "install.sh", bento_path=bento__path) %}
 {% set __pip_cache__ = common.mount_cache("/root/.cache/pip") %}
-{{ common.run_script(__install_python_scripts__, "install.sh to install python packages", run_args=__pip_cache__) }}
+# install python packages with install.sh
+RUN {{ __pip_cache__ }} bash -euxo pipefail {{ __install_python_scripts__ }}
 
 {% set __setup_script__ = expands_bento_path("env", "docker", "setup_script", bento_path=bento__path) %}
-{{ common.run_script(__setup_script__, "user setup scripts") }}
+RUN bash <<EOF
+set -euxo pipefail
+
+if [ -f {{ __setup_script__ }} ]; then
+    echo "Installing custom setup script..."
+    bash {{ __setup_script__ }}
+fi
+EOF
```

Comment:
Didn't we discuss this needed to be just 
```suggestion
RUN {{ __setup_script__ }}
```

We should probably do validation of the setup script on bento build; maybe we can chmod there.